TRANSLATING.md

```diff
@@ -240,3 +261,24 @@ Some of our English page content is generated from outside sources, and must not
 However, these pages are translated directly here and **these warnings are not meant for translations**.
 
 For these generated pages (like `configuration-reference.md`), we recommend **ignoring and removing the note and component (including its import) from the file**, thus avoiding confusion for other translators thinking that this warning applies to translations as well.
+
+## Review Tips
```

Comment:
Thinking about priorities again, maybe we want to move this even _above_ Translation Tips?

Reply:
Moved!